src/Common/ORDataObject/ORDataObject.php

```diff
@@ -138,7 +140,7 @@ function _load_enum($field_name, $blank = true)
         }
     }
 
-    function _utility_array($obj_ar, $reverse = false, $blank = true, $name_func = "get_name", $value_func = "get_id")
+    protected function _utility_array($obj_ar, $reverse = false, $blank = true, $name_func = "get_name", $value_func = "get_id")
```

Comment:
Got a white screen of death at Administration->Pratice->Practice Settings->Insurance Companies->Add a Company:
php error log
```
PHP Fatal error:  Uncaught Error: Call to protected method OpenEMR\\Common\\ORDataObject\\ORDataObject::_utility_array() from context 'C_InsuranceCompany' in /var/www/localhost/htdocs/openemr/controllers/C_InsuranceCompany.class.php:37\nStack trace:\n#0 /var/www/localhost/htdocs/openemr/library/classes/Controller.class.php(139): C_InsuranceCompany->edit_action()\n#1 /var/www/localhost/htdocs/openemr/controllers/C_PracticeSettings.class.php(59): Controller->act(Array)\n#2 /var/www/localhost/htdocs/openemr/library/classes/Controller.class.php(139): C_PracticeSettings->insurance_company_action('edit')\n#3 /var/www/localhost/htdocs/openemr/controller.php(5): Controller->act(Array)\n#4 {main}\n  thrown in /var/www/localhost/htdocs/openemr/controllers/C_InsuranceCompany.class.php on line 37, referer: http://localhost:8300/controller.php?practice_settings&insurance_company&action=list
```
(can solve this issue by making this function public instead of protected)

Reply:
Done 👍